tests/testthat/helper-orderly-runner.R

```diff
@@ -46,20 +36,20 @@ skip_if_no_redis <- function() {
 }
 
 
-test_prepare_orderly_example <- function(examples, ...) {
-  tmp <- tempfile()
-  withr::defer_parent(unlink(tmp, recursive = TRUE))
-  suppressMessages(orderly2::orderly_init(tmp, ...))
-  copy_examples(examples, tmp)
-  as.character(fs::path_norm(tmp))
+test_prepare_orderly_example <- function(examples, ..., env = parent.frame()) {
+  path <- withr::local_tempdir(.local_envir = env)
+  suppressMessages(orderly2::orderly_init(path, ...))
+  copy_examples(examples, path)
+
+  helper_add_git(path, orderly_gitignore = TRUE)
+  path
 }
 
 
-test_prepare_orderly_remote_example <- function(examples, ...) {
-  path_remote <- test_prepare_orderly_example(examples, ...)
```

Comment:
There was a bug here, where the remote directory would actually get cleaned up at the end of this function, before the caller could even use it.